CONTRIBUTING.md

```diff
@@ -1,49 +1,30 @@
-# Guidelines for Contributing Code
+# Contributing
 
-At New Relic we welcome community code contributions, and have taken effort to
-make this process easy for both contributors and our development team.
+Contributions are always welcome. Before contributing please read the
+[code of conduct](./CODE_OF_CONDUCT.md) and [search the issue tracker](issues); your issue may have already been discussed or fixed in `main`. To contribute,
+[fork](https://help.github.com/articles/fork-a-repo/) this repository, commit your changes, and [send a Pull Request](https://help.github.com/articles/using-pull-requests/).
 
-## Branches
+Note that our [code of conduct](./CODE_OF_CONDUCT.md) applies to all platforms and venues related to this project; please follow it in all your interactions with the project and its participants.
 
-The head of master will generally have New Relic's latest release. However,
-New Relic reserves the ability to push an edge to the master. If you download a
-release from this repo, use the appropriate tag. New Relic usually pushes beta
-versions of a release to a working branch before tagging them for General
-Availability.
+## Feature Requests
 
-## Testing
+Feature requests should be submitted in the [Issue tracker](../../issues), with a description of the expected behavior & use case, where they’ll remain closed until sufficient interest, [e.g. :+1: reactions](https://help.github.com/articles/about-discussions-in-issues-and-pull-requests/), has been [shown by the community](../../issues?q=label%3A%22votes+needed%22+sort%3Areactions-%2B1-desc).
+Before submitting an Issue, please search for similar ones in the
+[closed issues](../../issues?q=is%3Aissue+is%3Aclosed+label%3Aenhancement).
 
-The code includes a suite of unit tests with each package which should be used
-to verify your changes don't break existing functionality.
+## Pull Requests
 
-### Running Tests
```

Comment:
I would keep that section

Reply:
Same as with mysql, test section is now in the README